modules/claims_api/spec/requests/v1/power_of_attorney_request_spec.rb

```diff
@@ -202,6 +209,16 @@
           end
         end
       end
+
+      context 'when a non-accredited representative and non-veteran request active power of attorney' do
```

Comment:
similar question as above. since this only validates non-veteran requests, do we need to worry about adding `/active/` tests for veteran requests? I don't think there's any in this file, so the only _new_ test I added was a specific failing case for the accredited validation

Reply:
Yeah, I mean, for v1 specifically we should be validating that the Veteran making the request is the same Veteran as the subject of the request. For v0...this doesn't apply, since Veterans and representatives can't use v0.